docs/diagnostics/PX1085.md

```diff
@@ -0,0 +1,35 @@
+# PX1085
+This document describes the PX1085 diagnostic.
+
+## Summary
+
+| Code   | Short Description                                                                                     | Type                           | Code Fix    | 
```

Comment:
The type of the diagnostic is Error, not Warning

Reply:
@metallmonster , it is marked with ISVCertW1 in jira - that is, it was supposed to be a warning.

Reply:
@k-s-popova I've changed severity of the diagnostic. The changes are in the following pull-request https://github.com/Acumatica/Acuminator/pull/158